src/KubernetesClient/IntOrStringConverter.cs

```diff
@@ -21,7 +21,7 @@ public override void Write(Utf8JsonWriter writer, IntstrIntOrString value, JsonS
         {
             var s = value?.Value;
 
-            if (int.TryParse(s, out var intv))
+            if (long.TryParse(s, out var intv))
```

Comment:
From the code<https://github.com/kubernetes-client/csharp/blob/01b2aa832cfdc6f2c89153fe0123edc0a13b1bbf/src/KubernetesClient/IntstrIntOrString.cs#L11-L14>, we could see `long` is acceptable, so maybe we should use `long.TryParse`